src/test/scala/chisel3/tests/AsyncResetRegTest.scala

```diff
@@ -0,0 +1,159 @@
+// See LICENSE for license details.
+
+package chisel3.tests
+
+
+import chisel3._
+import chisel3.core.IntParam
+import chisel3.tester._
+import chisel3.util.{Cat, HasBlackBoxResource}
+import firrtl.ir.{Param, Type}
+import org.scalatest._
+import treadle.executable.{PositiveEdge, Transition}
+import treadle.{ScalaBlackBox, ScalaBlackBoxFactory}
+
+object AsyncResetReg {
+  val bitWidth: Int = 3
```

Comment:
wat? Would it ever not be one?

Reply:
Simple way to define constants for tests that might change in future. Removed for now